src/julia.h

```diff
@@ -2289,6 +2289,16 @@ typedef struct _jl_task_t {
     // uint8_t padding1;
     // multiqueue priority
     uint16_t priority;
+    // flag indicating whether or not this to measure the wall and cpu time of this task
+    uint8_t is_timing_enabled;
```

Comment:
```suggestion
    uint8_t timing_enabled;
```
If accepted, there are other places to change that I haven't commented -- will need a search/replace.